grow/translators/google_sheets.py

```diff
@@ -54,13 +55,12 @@ def _download_sheet(self, spreadsheet_id, locale):
                 raise translator_errors.NotFoundError(
                     'Translation for {} not found.'.format(locale))
             raise
-
         return resp['values']
 
     def _download_content(self, stat):
         spreadsheet_id = stat.ident
         values = self._download_sheet(spreadsheet_id, stat.lang)
-        source_lang, lang = values.pop(0)
+        source_lang, lang, _ = values.pop(0)
```

Comment:
Just a note, no action needed – I wonder if we should make this accept variable numbers of columns to increase the robustness. Again, no action needed but this would require updating if future enhancements make changes to the columns.